apps/financial-aid/web-veita/src/routes/MySettings/MySettings.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useContext, useState } from 'react'
+
+import {
+  ApplicationOverviewSkeleton,
+  LoadingContainer,
+} from '@island.is/financial-aid-web/veita/src/components'
+import {
+  Text,
+  Box,
+  Input,
+  ToggleSwitchCheckbox,
+} from '@island.is/island-ui/core'
+import { AdminContext } from '@island.is/financial-aid-web/veita/src/components/AdminProvider/AdminProvider'
+
+import * as styles from '@island.is/financial-aid-web/veita/src/components/Profile/Profile.css'
+import { InputType } from '@island.is/financial-aid-web/veita/src/components/Profile/EmployeeProfile'
+import { isEmailValid } from '@island.is/financial-aid/shared/lib'
+
+interface mySettingsState {
+  nationalId?: string
+  name?: string
+  email?: string
+  nickname?: string
+  hasError: boolean
+  hasSubmitError: boolean
+}
+
+export const MySettings = () => {
```

Comment:
Question, this component doesn't save changes so it is only displaying values correct?

Reply:
Yesss, I wanted to save that for another PR

Reply:
Cool, cool